src/naive/internals.rs

```diff
@@ -687,33 +691,6 @@ mod tests {
         }
     }
 
-    #[test]
-    fn test_mdf_to_of() {
```

Comment:
I ported a couple of tests for cases that would otherwise not be covered to `naive::date::tests`.
But not entirely 1:1. For example I see no added value for `test_mdf_to_of` when there also was `test_mdf_to_of_to_mdf` wich does the same thing and adds one extra step.